prisma/schema.prisma

```diff
@@ -13,11 +13,54 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-
 model User {
   id        String   @id @default(uuid()) @db.Uuid
   email     String   @unique
-  name      String?
+  image_url String?
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
-}
\ No newline at end of file
+
+  // Relations
+  Account   Account[]
+}
+
+model Account {
```

Comment:
- A single account will be shared with multiple users 
- Use AccountUser table as a pivot table for account & User

Reply:
![Blank diagram (2)](https://github.com/user-attachments/assets/dc9b91b1-fad1-4a51-84e8-cd710b5dfa8a)


I have revised the ER diagram and implemented the changes requested.